### PR TITLE
LV/Legality: fix style after cursory reading (NFC)

### PR DESCRIPTION
**Pull request by artagnon**

### CODE REVIEW

File: llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp [hunk 1]
```diff
@@ -216,20 +216,18 @@ void LoopVectorizeHints::emitRemarkWithHints() const {
...
-      if (Force.Value == LoopVectorizeHints::FK_Enabled) {
-        R << " (Force=" << NV("Force", true);
-        if (Width.Value != 0)
-          R << ", Vector Width=" << NV("VectorWidth", getWidth());
-        if (getInterleave() != 0)
-          R << ", Interleave Count=" << NV("InterleaveCount", getInterleave());
-        R << ")";
-      }
-      return R;
+    OptimizationRemarkMissed R(LV_NAME, "MissedDetails", TheLoop->getStartLoc(),
```
Comment (nikic) [inline llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp:219]: Add empty line above here?

File: llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp [hunk 2]
```diff
@@ -1154,8 +1145,7 @@ bool LoopVectorizationLegality::canVectorizeMemory() {
     }
   }
 
-  PSE.addPredicate(LAI->getPSE().getPredicate());
-  return true;
+  return PSE.addPredicate(LAI->getPSE().getPredicate()), true;
```
Comment (nikic) [inline llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp:1148]: IMHO this was much clearer before. Let's please not use comma expressions...

**Review by fhahn — COMMENTED** [review]
Would be good to update the title to be clear this fixes style issues

**Review by nikic — APPROVED** [review]
LGTM

**Review by fhahn — APPROVED** [review]
LGTM, thanks!

**Pull request merged**